### PR TITLE
Check in breeze theme

### PR DESCRIPTION
**Pull request by knuton**
Checks in the breeze theme and adds appropriate licensing information.

## Checklist

-   <del>[ ] Changelog updated</del>
-   [x] Code documented

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -76,9 +76,11 @@ Update the change log for every release. See http://keepachangelog.com/ for form
 - [Buildroot](https://buildroot.org/): Builder for embedded Linux systems. Also not very well suited for desktop functionality.
 - [not-os](https://github.com/cleverca22/not-os): A NixOS based system generator. Much more minimal than NixOS, does not use systemd and is not compatible with existing NixOS modules.
 
-## Attribution
+## Attribution and Licensing
 
-This software contains portions from other open-source projects.
+Most code in this repository is authored by the Dividat AG and the project contributors. This code is licensed under an MIT license.
+
+Some source files in this project are portions of other open-source project and may be released under different licenses. The applicable licenses are stated here as well as in the relevant subdirectories.
```
Comment (guyonvarch) [inline Readme.md:83]: > The applicable licenses are stated here

Should we add the licence of `breeze` here then ?

Comment (knuton) [reply Readme.md:83]: Yes, already added: https://github.com/dividat/playos/pull/48/files#diff-799de429c9fec9cf73e3c43693aea5c9d8a1dbbfd614e3d1c0e6453e892d0023

Comment (guyonvarch) [reply Readme.md:83]: Nitpicking: should we replace “and” with “or” then ?

Comment (knuton) [reply Readme.md]: Ah, now I understand the misunderstanding. Yes: 281cc86bd85ba14f902b62f4e9e3efae40af6fe4

**Comment by guyonvarch** [comment]
The cursor theme is available again:
https://code.jpope.org/jpope/breeze_cursor_sources/raw/master/breeze-contrast-cursor-theme.zip

But we’re making sure we don’t get a 404 again by including the files here.

I don’t have the cursor theme loaded using the vm (`make vm`).



I checked I have it on `develop`:

**Comment by knuton** [comment]
> But we’re making sure we don’t get a 404 again by including the files here.

Yes, I'd rather do that after having run into the issue once.

> I don’t have the cursor theme loaded using the vm (`make vm`).

Apologies, I must have messed it up before committing. Updated the PR with a re-tested version.

**Review by guyonvarch — APPROVED** [review]
Working as expected!

**Pull request merged**